cygnus-ngsi-ld/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.telefonica.iot</groupId>
     <artifactId>cygnus-ngsi-ld</artifactId>
-    <version>2.18.0</version>
+    <version>2.18.0-SNAPSHOT</version>
```

Comment:
cygnus-common entry should be also included

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2167/commits/15f06b3eb0f03c28f38be5a1b3cbd5deffffe15c